.github/workflows/update_versions.yml

```diff
@@ -0,0 +1,47 @@
+# This is a basic workflow that is manually triggered
+
+name: Update Versions
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      version:
+        description: 'New version to use for the extension. Example: 1.5.2'
+        required: true
+
+      branch:
+        description: 'Branch to be used when updating versions'
+        required: true
+
+      core-dependency:
+        description: 'If a version is provided, update AEPCore dependency in podspec and Package.swift'
+        required: false
```

Comment:
Add a similar input for Identity, which is listed as a dependency in the Podspec.